db/stations.js

```diff
@@ -50,8 +50,21 @@ exports.levenLookup = function(needle, limit, callback) {
     // itterate over stations to find lowest levenshtein distance string
     _.forOwn(stations, function(id, station) {
       var score = leven(needle, station);
-      if (score < 5 ) // if the score is larger than 5, it's probably nonsense
-        results.push({ score: score, id: parseInt(id), station: station });
+
+      // manipulate the score for stations beginning with the specified needle
+      // e.g gent -> gent-sint-pieters, gent-dampoort, ...
+      if (station.indexOf(needle) === 0) {
+        score = 0;
+      }
+
+      // if the score is larger than 5, it's probably nonsense
+      if (score < 5 ) {
```

Comment:
It probably makes more sense to return from the iteration when the score is larger than 5.

```javascript
var score = leven(needle, station);
if (score > 5 ) // if the score is larger than 5, it's probably nonsense
  return;
```